citations-tool/tool/src/webapp/vm/citation/_listNestedCitations.vm

```diff
@@ -211,12 +213,12 @@
 					## else if it's a h1 citation
 					#elseif ($h2Section.getSectiontype()=='CITATION')
 					## nested citations
-						<ol id="$addSubsectionId" class="h4NestedLevel holdCitations">
 							#set( $citation = $allCitationCollection.getCitation($h2Section.getCitationid() ))
+							<li id='$linkId' data-citationId='$h3Section.getCitationid()' data-location='$num' data-sectiontype='CITATION'>
```

Comment:
This looks wrong, you've got 2 opening `<li>`s but only one closing one. Also you're mixing `h2Section` and `h3Section` which doesn't seem correct.